CMakeLists.txt

```diff
@@ -27,3 +27,12 @@ set(SOURCES
 add_subdirectory(lib/pybind11)
 pybind11_add_module(core ${SOURCES} "${SOURCE_DIR}/python_bindings.cpp")
 install(TARGETS core DESTINATION src/pinetree)
+
+SET(TEST_DIR "tests")
+SET(TESTS ${SOURCES}
+    "${TEST_DIR}/test_main.cpp"
+    "${TEST_DIR}/unit_tests.cpp")
+
+# Generate a test executable
+#include_directories(lib/catch/include)
+add_executable("${PROJECT_NAME}_test" ${TESTS})
```

Comment:
Further down, it looks to me like you hardcode "pinetree_test". Should you do the same here, to make sure things are consistent? I assume if somebody changed the project name things would break because of the hardcoding further down, right? I generally prefer not to hardcode, but in this particular case it seems fine.

Reply:
It does't look like "pinetree_test" is hardcoded anywhere in CMakeLists but it IS hardcoded in setup.py because it doesn't have a project variable. Should I just create a "PROJECT_NAME" variable in setup.py as well? Ultimately you would still need to remember to change both files but I think having the consistency is a little nicer.

Reply:
Yes, that sounds good to me, thanks!